packages/core/src/components/pane/pane.tsx

```diff
@@ -0,0 +1,620 @@
+/*
+ * SPDX-FileCopyrightText: 2023 Siemens AG
+ *
+ * SPDX-License-Identifier: MIT
+ *
+ * This source code is licensed under the MIT license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import {
+  Component,
+  Element,
+  Event,
+  EventEmitter,
+  h,
+  Host,
+  Prop,
+  State,
+  Watch,
+} from '@stencil/core';
+import anime from 'animejs';
+import Animation from '../utils/animation';
+import { applicationLayoutService } from '../utils/application-layout';
+import { matchBreakpoint } from '../utils/breakpoints';
+
+export type Composition = 'top' | 'left' | 'bottom' | 'right';
+export type ExpandedChangedEvent = {
+  paneId: string;
+  expanded: boolean;
+};
+export type SlotChangedEvent = {
+  paneId: string;
+  newSlot: string;
+};
+export type HideOnCollapseChangedEvent = {
+  paneId: string;
+  hideOnCollapse: boolean;
+};
+export type VariantChangedEvent = {
+  paneId: string;
+  variant: 'floating' | 'inline';
+};
+export type BorderlessChangedEvent = {
+  paneId: string;
+  borderless: boolean;
+};
+
+let idCounter = 0;
+
+/**
+ * @since 2.1.0
+ */
+@Component({
+  tag: 'ix-pane',
+  styleUrl: 'pane.scss',
+  shadow: true,
+})
+export class Pane {
+  @Element() hostElement: HTMLIxPaneElement;
+
+  /**
+   * Title of the side panel
+   */
+  @Prop() heading: string = 'Pane title';
+
+  /**
+   * Variant of the side pane.
+   * Defaults to the variant attribute of the pane layout. If used standalone it defaults to inline.
+   */
+  @Prop() variant: 'floating' | 'inline';
+
+  /**
+   * Define if the pane should have a collapsed state
+   */
+  @Prop() hideOnCollapse: boolean = false;
+
+  /**
+   * The maximum size of the sidebar, when it is expanded
+   */
+  @Prop() size:
+    | '240px'
+    | '320px'
+    | '360px'
+    | '480px'
+    | '600px'
+    | '33%'
+    | '50%' = '240px';
+
+  /**
+   * Toggle the border of the pane.
+   * Defaults to the borderless attribute of the pane layout. If used standalone it defaults to false.
+   */
+  @Prop() borderless: boolean;
+
+  /**
+   * State of the pane
+   */
+  @Prop({ mutable: true }) expanded: boolean = false;
+
+  /**
+   * Placement of the sidebar
+   */
+  @Prop({ mutable: true }) composition: Composition = 'top';
```

Comment:
Kthrin and I chose the term because other property names (for example: position, placement, …) would also imply positioning of the rogue pane (pane which is used outside of a layout), which is technically not feasible.